client/verta/verta/registry/entities/_modelversion.py

```diff
@@ -580,7 +580,7 @@ def log_artifact(self, key, artifact, overwrite=False, _extension=None):
         self._update(self._msg, method="PUT")
         self._upload_artifact(key, artifact_stream, artifact_type=artifact_type)
 
-    # TODO: Fix for VR-12591
+
```

Comment:
It looks like this PR introduces just a couple of extra newlines. PEP8 recommends just one blank line to separate methods in a class.